convert_state.py

```diff
@@ -1,60 +1,58 @@
+"""Convert between state name and abbreviation"""
+import json  # import json for json processing
 
 
-
-import json  # import json for json processing
 def main():
+
+    """ Main function creates dictionary of states and runs menu """
+
     with open('data.json','r') as file:
         data = file.read()
-        data = json.loads(data)
+        state_data = json.loads(data)
 
-    print(data)
-    
-    # Create a dictionary of states -> abbreviations, and a dictionary of abbreviations -> states 
-    stateAbbr = data #dictionary of state abbreviations keys and state name values 
-    stateAbbr2 = {}# dictionary of state name keys and state abbreviation values
-    # state key is the key in the state dictionary. state value is the value in the dictionary 
-    for statekey, statevalue in stateAbbr.items() :
-        stateAbbr2[statevalue] =  statekey
+    state_key_and_name = state_data 
+    state_key_and_abbrev = {}
 
+    for statekey, statevalue in state_key_and_name.items() :
+        state_key_and_abbrev[statevalue] = statekey
 
-
-    #loop
     while True:
         print('1. Convert state to abbreviation')
-        print("2. Convery abreviasion to state")
-        print(' 3. quit')
-        choice =   input("Enter choice : ")
+        print("2. Convert abbreviation to state")
+        print('3. Quit')
+        choice = input("Enter choice:")
 
         if choice=="1":
```

Comment:
Single quotes for all strings, put a space before and after the == operator `if choice == '1':`